### ISSUE TITLE
validate_certs_disable effectively defaults to false contrary to documentation

### ISSUE THREAD

**Issue by scoffeyfv** [F5Networks/terraform-provider-bigip#716]
From v1.15.2 the provider introduced device TLS certificate validation along with the parameter **validate_certs_disable** to allow for disabling this.  According to documentation this defaults to 'enable' (i.e. validation is disabled) but the actual default behaviour behaviour is as per enabled,  i.e. certs are validated,  for example with a default selfsigned BIG-IP cert and no configured attribute/variable I see 

Post "https://172.16.27.101/mgmt/shared/authn/login": x509: “localhost.localdomain” certificate is not standards compliant

I believe the issue lies on the schema map:
```
 "validate_certs_disable": {
   Type:        schema.TypeBool,
   Optional:    true,
   Default:     true,
   Description: "Enables TLS certificate check on BIG-IP",
   DefaultFunc: schema.EnvDefaultFunc("BIGIP_VERIFY_CERT_DISABLE", nil)
```

- according to Hashicorp,  it is not valid to use Default and DefaultFunc together
   https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#defaultfunc
- irrespective,  if DefaultFunc over-rides Default, then the default value of "nil" implies boolean -false-.  I suspect DefaultFunc wins, so we get the default of false -> validates certs.

Also: the Description should read "**Disables** TLS certificate check" to match the function.

**Comment by RavinderReddyF5** [comment]
@scoffeyfv did u run into any issue ? we didnot see any issue with default behaviour of disable cert validation. if you need to validate certs please set this parameter value to `false`. and yes Description/DOC needs correction it should be **Disables** TLS certificate check.

**Comment by RavinderReddyF5** [comment]
Hi @scoffeyfv , please verify your behaviour on v1.16.0, and let us know if you still see any issue

**Comment by scoffeyfv** [comment]
Hi @RavinderReddyF5 - a customer and I saw certificate errors when upgrading to 1.15.2.  I checked the code and spotted the questionable defaults.  However what I admit we did not do yesterday was test with validate_certs_disable explicitly set to true.   Even with that set to true (or false), we have cert errors.   I just saw issue #720 arrive, where they report setting token auth to false fixes the issue.   I can confirm that my customer and I both have token auth set to true.  In my environment, setting it to false indeed removes the cert errors.  Maybe we are trying to fetch a token before setting TLSClientConfig.InsecureSkipVerify ?
